{"uuid": "fea2b939-7fc2-4577-9593-67790df3f299", "children": ["83b6336c-50f1-4d2c-b387-cea383dd4f5e"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'\n", "trace": "  File \"/home/test/Desktop/sprint_7/venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/test/Desktop/sprint_7/venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/test/Desktop/sprint_7/venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/test/Desktop/sprint_7/conftest.py\", line 22, in driver\n    driver = webdriver.Firefox(executable_path=GeckoDriverManager().install())\n"}, "start": 1728419158714, "stop": 1728419166778}], "start": 1728419158714, "stop": 1728419166974}